doc/source/how-to/documenting.rst

```diff
@@ -260,6 +260,35 @@ Because these examples are built using Sphinx-Gallery, you must follow its codin
 
 :ref:`General example` uses Python and Sphinx-Gallery.
 
+Generate documentation without examples
++++++++++++++++++++++++++++++++++++++++
+
+The documentation for some PyAnsys repositories is built using Sphinx-Gallery and
+includes examples generated with the assistance of Ansys products.
+Building documentation locally is time consuming in these cases,
+particularly if you need to test changes only in the documentation excluding the examples.
+
+To generate documentation without examples, use command ``make html-noplot`` with
+appropriate changes in ``make.bat`` and ``Makefile``. Change needed for ``Makefile``
+is mentioned below. Refer `sphinx-gallery documentation 
+<https://sphinx-gallery.github.io/stable/configuration.html#building-without-executing-examples>`_.
+
+.. code:: shell
```

Comment:
making this as a tab with makefile and make.bat file would be helpful

Reply:
I completely agree